main-app/app/scripts/controller.js

```diff
@@ -1,10 +1,10 @@
 
 (function () {
     'use strict';
-    angular.module('tombola.snapCardGame')
+    angular.module('Tombola.SnapCardGame')
 
-        .controller('snapCardController',['$scope','gameModel',function ($scope, gameModel){
-
-            $scope.gameModel = gameModel;
+        .controller('SnapCardController', ['$scope', 'GameModel', 'DeckService', function ($scope, GameModel, DeckService){
```

Comment:
Bad news! 
    function ($scope, GameModel, DeckService)
Should be
    function ($scope, gameModel, deckService)
Correct naming of variables - I'm suprised your linter is passing - the strings are fine & as we've used the explict naming syntax (with the array) it'll work